### PR TITLE
Start of carbon style guide integration

### PR DESCRIPTION
**Pull request by donaldkibet**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,6 +77,9 @@
     "webpack-dev-server": "^3.11.0"
   },
   "dependencies": {
+    "carbon-components": "^10.19.0",
```
Comment (FlorianRappl) [inline package.json:80]: Should be in shared dependencies of the app shell.

File: src/carbon-esm-styleguide.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import "../node_modules/carbon-components/css/carbon-components.min.css";
```
Comment (FlorianRappl) [inline src/carbon-esm-styleguide.js:1]: I guess a better variant uses the index.html of the app shell - we should `<link>` the stylesheets instead imho.

**Review by FlorianRappl — COMMENTED** [review]
Maybe we should move these changes into the app shell?

cc @brandones

**Comment by brandones** [comment]
@FlorianRappl Seems reasonable to me. I had recommended that it go in esm-styleguide. My idea was that we would probably want some amount of machinery around Carbon—compiling & customizing SCSS, etc., which would be nice to have in its own space. Most (if not all) of what presently exists in esm-styleguide would go away.

However, since right now all we're doing with Carbon is importing the CSS globally, it seems fine to include it directly in the app shell.

**Review by FlorianRappl — APPROVED** [review]
Merged for now. Will be changed in esm-core.

**Pull request merged**